app/views/pages/governance/CPO.njk

```diff
@@ -1,33 +0,0 @@
-{% set pageTitle = "Corporate Portfolio Office" %}
```

Comment:
Do we need to delete this page from the `routes.js` file?

Reply:
pages/governance/cpo ? looks like this is still being used.

Reply:
I have asked if this is still relevant.